### ISSUE TITLE
Adhafera weather icons do not work

### ISSUE THREAD

**Issue by NeoFax75** [closebox73/Leonis#5]
I have the feather font installed, the wether.json file is created, but all I get is a square box instead of a clear sky icon.

**Comment by closebox73** [comment]
are the font is from my config?
update font cache? execute fc-cache - fv
then reload conky

**Comment by closebox73** [comment]
Ah.. Sorry I know the problem
open Adhavera.conf see line 76 and change `Material` to `Feather`

**Comment by NeoFax75** [comment]
Nope, now I get no Wi-Fi icon and no Weather icon and it changed all of the fonts smaller which moved evrything around so it does not line-up.  Here is a copy of my config:

'''
conky.config = {
--==============================================================================

--  2021/06/12
--  this theme is for conky version 1.10.8 or newer
-- 
--  ADHAFERA
--  ( A part of Leonis Conky themes pack )
--
--  author  : Closebox73
--  license : Distributed under the terms of GPLv3
--  notes   : Created on 1366x768 Monitor

-- Update :
-- 26 June 22 >>  i change music player from MOCP to MPD

--==============================================================================

-- Size and Position settings --
  alignment = 'top_left',
  gap_x = 20,
  gap_y = 50,
  maximum_width = 320,
  minimum_height = 600,
  minimum_width = 320,
  
-- Text settings --
  use_xft = true,
  font = 'Roboto:light:size=9',
  
-- Color Settings --
  default_color = 'white',
  default_outline_color = 'white',
  default_shade_color = 'white',
  color1 = '962AC3',
  
-- Window Settings --
  background = false,
  border_width = 1,
  draw_borders = false,
  draw_graph_borders = false,
  draw_outline = false,
  draw_shades = false,
  own_window = true,
  own_window_colour = '000000',
  own_window_class = 'Conky',
  own_window_argb_visual = true,
  own_window_type = 'desktop',
  own_window_transparent = true,
  own_window_hints = 'undecorated,below,sticky,skip_taskbar,skip_pager',
  stippled_borders = 0,
  
-- Others --
  cpu_avg_samples = 2,
  net_avg_samples = 2,
  double_buffer = true,
  out_to_console = false,
  out_to_stderr = false,
  extra_newline = false,
  update_interval = 1,
  uppercase = false,
  use_spacer = 'none',
  show_graph_scale = false,
  show_graph_range = false,
}

conky.text = [[
${execi 100 ~/.config/conky/Adhafera/scripts/weather.sh}\
${execi 100 ~/.config/conky/Adhafera/scripts/weather-icon.sh white3 $(cat ~/.cache/weather.json | jq -r '.weather[0].icon')}\
${image ~/.config/conky/Adhafera/res/box.png -s 288x432 -p 0,50}\
${offset 100}${voffset 10}${color}${font Bebas Neue:size=20}${time %M}${font}
${offset 30}${voffset -25}${color}${font Bebas Neue:size=60}${time %H}${font}
${alignr 62}${voffset -22}${color}${font Bebas Neue:size=12}${execi 3 ~/.config/conky/Adhafera/scripts/Greeting.sh}, ${execi 600 whoami}${font}
${offset 25}${voffset 153}${color1}${font Bebas Neue:bold:size=22}${time %d}${goto 168}Uptime${font}
${offset 25}${voffset 0}${color1}${font Nunito:size=10}${time %a - %b}${goto 168}${Uptime}${font}
${offset 105}${voffset 35}${color}${font Feather:size=19}${execi 3 ~/.config/conky/Adhafera/scripts/wifi-icon}${goto 245}${font Feather:size=20}${execi 15 ~/.config/conky/Adhafera/scripts/weather-text-icon}
${offset 25}${voffset 60}${color1}${font Bebas Neue:bold:size=22}Wi-Fi${goto 168}${execi 100 cat ~/.cache/weather.json | jq -r '.weather[0].main' | sed "s|\<.|\U&|g"}${font Bebas Neue:size=0}
${offset 25}${voffset 3}${color1}${font Nunito:size=10}${execi 3 ~/.config/conky/Adhafera/scripts/ssid}${goto 168}${execi 100 cat ~/.cache/weather.json | jq '.main.temp' | awk '{print int($1+0.5)}'}°F - ${execi 100 cat ~/.cache/weather.json | jq -r '.name'}${font Bebas Neue:size=0}
${offset 25}${voffset 40}${if_running audacious}${font Nunito:size=12}${color1} *   ${color1}Now ${exec audtool --playback-status}  ${color}"${exec audtool --current-song}"   \
${font Bebas Neue:size=24}${color1}Artist  ${color}${exec audtool --current-song-tuple-data artist}   \
${color1}Album  ${color}${exec audtool --current-song-tuple-data album}   \
${color1}Track no.  ${color}${exec audtool --current-song-tuple-data track-number}   \
${color1}Elapsed  ${color}${exec audtool --current-song-output-length} | ${exec audtool --current-song-length}   *${else}${color1}Audacious is not running right now. ${color}Shall we play a song? \
${offset 25}${voffset 4}${color}${if_running audacious}${font Material:size=12} ${font Nunito:size=12}$"${exec audtool --current-song}"${else} ${endif}
]]
'''

**Comment by NeoFax75** [comment]
I figured out the problem by using another conky template of yours.  It is the weather-text-icon file.  The file included in the Zozma theme pack has the correct conversion text.
